### DISCUSSION TITLE
Is there a way I can include code examples on my repo's webpage like Git Hub does?

### DISCUSSION THREAD

**Discussion by DataJuggler** [community/community#38714] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have some example code in one of my projects Git Hub page. I would like to add some example code without reinventing the wheel (again) and writing my own styling code.

Are there any components or shared ways to add code snippets to a C# Blazor site the way Git Hub does?

I get more visitors to the product page then I do for Git Hub, probably from the Nuget package links.

Thanks

Corby / Data Juggler

**Comment by ossanna16** [comment]
Hi there @DataJuggler and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
